.kokoro/release/publish.cfg

```diff
@@ -0,0 +1,9 @@
+build_file: "google-cloud-cpp/.kokoro/release/publish.sh"
```

Comment:
Generally we keep all our ci scripts for Kokoro in `ci/kokoro/` can we move these there?